rpi/default.nix

```
@@ -304,14 +320,8 @@ in
       kernelParams =
         if cfg.uboot.enable then [ ]
         else [
-          # This is ugly and fragile, but the sdImage image has an msdos
-          # table, so the partition table id is a 1-indexed hex
-          # number. So, we drop the hex prefix and stick on a "02" to
-          # refer to the root partition.
-          "root=PARTUUID=${lib.strings.removePrefix "0x" config.sdImage.firmwarePartitionID}-02"
-          "rootfstype=ext4"
-          "fsck.repair=yes"
-          "rootwait"
+          "console=tty1"
+          "console=serial0,115200n8"
```

This comment wasn't moved.
```suggestion
          # https://github.com/raspberrypi/firmware/issues/1539#issuecomment-784498108
          "console=tty1"
          "console=serial0,115200n8"
```